### ISSUE TITLE
Offline Use

### ISSUE THREAD

**Issue by apaly** [Gumphrie/zebkit-theme-builder#1]
Hi,
I would like to use the Zebkit Theme Builder offline, that is without an HTTP server. Is that possible and could you please provide a file for this? This would be very much appreciated because I am using Zebkit offline.

Thanks

**Comment by Gumphrie** [comment]
Hi,
Not really, as it uses a node web server to do the packaging for you (see https://github.com/Gumphrie/zebkit-theme-builder/blob/master/setup/compress.js) to create the appropriate zebkit.min.js, etc.. But you don't need to have an already existing web server, (if that's what you mean?), just build the dependencies and it will create one for you.

**Comment by apaly** [comment]
Hi,
Thanks for your reply.
I am not a web technology savvy person, I don't have node.js installed and neither have I used it before. That is why I want a repackaged file that I can use offline without a web server. This is exactly how I am using ZebKit at the moment. I just include ZebKit in my HTML page and work with it totally offline - without a server.

I hope my explanation is clearer? Could please help with the prepackaged file?

Thanks.

**Comment by Gumphrie** [comment]
So you don't want to build your own theme and just want the default one created in the project in one zebkit.min.js?

**Comment by apaly** [comment]
Yes.

**Comment by Gumphrie** [comment]
[zebkit.min.js.gz](https://github.com/Gumphrie/zebkit-theme-builder/files/5232689/zebkit.min.js.gz)
OK, should be attached

**Comment by apaly** [comment]
Hi,
Thanks a lot.
Sorry to bother you again could you also provide a non minified version - zebkit.js?

**Comment by Gumphrie** [comment]
Not really, the non minified version doesn't include the other files you need

**Comment by apaly** [comment]
Okay,
Is it possible for me to manually link your non minfied version to Zebkit?
If so, could you please provide your prepackaged non minified version and an explanation on how I could link it manunally to ZebKit?
Thanks.
